sunpy/instr/goes.py

```diff
@@ -426,6 +427,12 @@ def _goes_chianti_tem(longflux: u.W/u.m/u.m, shortflux: u.W/u.m/u.m, satellite=8
     return temp, em
 
 
+@manager.require('FILE_TEMP_COR',
```

Comment:
I don't think these names should be all caps, that's shorthand for constant and they aren't here.